src/nixos.rs

```diff
@@ -129,6 +145,7 @@ impl OsRebuildActivateArgs {
 
     let message = match variant {
       BuildVm => "Building NixOS VM image",
+      BuildImage => "Building Nixos Image",
```

Comment:
```suggestion
      BuildImage => "Building NixOS image",
```

For anyone with a working set of eyes, this will look inconsistent. There is also no need to capitalize the I in "image".